frontend/pages/DashboardPage/cards/MDM/MDM.tsx

```diff
@@ -65,6 +65,28 @@ const EmptyMdmSolutions = (): JSX.Element => (
   />
 );
 
+type IMdmSolutionMap = Record<string, IMdmSolution>;
+
+const reduceSolutionsToObj = (mdmSolutions: IMdmSolution[]) => {
+  return mdmSolutions.reduce<IMdmSolutionMap>((acc, nextSolution) => {
```

Comment:
Do we care what happens to the `server_url`? If not it might be cleaner to pick the properties we care about into a subtype of IMdmSolution to avoid issues down the line where a caller relies on the accuracy of properties other than name and host count.

Reply:
I just passed the data through for simplicity, but I do like the suggestion of passing only the data needed for the Table. I will make that update.